src/components/home/modals/Account.vue

```diff
@@ -207,11 +207,11 @@
           <b-form-input readonly
                         size="sm"
                         class="input-t"
-                        :value="tagOfColdWallet[addr] ? pubkey : addr">
+                        :value="tagOfColdWallet[addr] ? pubkey.publicKey : addr">
```

Comment:
At Line 195, change variable name `pubkey` to `coldAccount`. And then change `pubkey.publicKey` to `pubkey. coldAccount` here